benchmark/helper_test.go

```diff
@@ -0,0 +1,170 @@
+package benchmark
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/bandprotocol/chain/v2/pkg/obi"
+	"github.com/bandprotocol/chain/v2/testing/testapp"
+	oracletypes "github.com/bandprotocol/chain/v2/x/oracle/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/tendermint/tendermint/abci/types"
+)
+
+type Account struct {
+	testapp.Account
+	Num uint64
+	Seq uint64
+}
+
+type BenchmarkCalldata struct {
+	DataSourceId uint64
+	Scenario     uint64
+	Value        uint64
+}
+
+func GetBenchmarkWasm() ([]byte, error) {
+	oCode, err := ioutil.ReadFile("./testdata/benchmark-oracle-script.wasm")
+	return oCode, err
+}
+
+func GenMsgRequestData(
+	sender *Account,
+	oracleScriptId uint64,
+	dataSourceId uint64,
+	scenario uint64,
+	value uint64,
+) []sdk.Msg {
+	msg := oracletypes.MsgRequestData{
+		OracleScriptID: oracletypes.OracleScriptID(oracleScriptId),
+		Calldata: obi.MustEncode(BenchmarkCalldata{
+			DataSourceId: dataSourceId,
+			Scenario:     scenario,
+			Value:        value,
+		}),
+		AskCount:   1,
+		MinCount:   1,
+		ClientID:   "",
+		FeeLimit:   sdk.Coins{sdk.NewInt64Coin("uband", 1)},
+		PrepareGas: GasLimit,
+		ExecuteGas: GasLimit,
```

Comment:
*Optional
maybe consider separate to PrepareGasLimit and ExecuteGasLimit for better understanding and more customization in the future

Reply:
ok. let me separate the variable for prepare and execute.